packages/dds/tree/src/simple-tree/api/schemaCompatibilityTester.ts

```diff
@@ -23,19 +22,20 @@ import {
 	type FieldDiscrepancy,
 } from "../discrepancies.js";
 import { toStoredSchema } from "../toStoredSchema.js";
+import type { TreeSchema } from "./configuration.js";
+import { isObjectNodeSchema } from "../node-kinds/index.js";
 
 /**
- * A collection of View information for schema, including policy.
+ * A collection of View information for schema
  * @remarks
  * This contains everything needed to determine compatibility with a given stored schema.
  */
 export class SchemaCompatibilityTester {
-	/**
-	 * @param viewSchemaRoot - Schema for the root field.
-	 */
 	public constructor(
-		public readonly policy: FullSchemaPolicy,
-		public readonly viewSchemaRoot: FieldSchema,
+		/**
+		 * Schema for the view
+		 */
+		public readonly viewSchema: TreeSchema,
```

Comment:
Take in a tree schema, so the map from identifier to TreeNodeSchema already exists instead of having to be made in here.

Also since simple-tree assumes default schema policy due to its hard coded set of field kinds, no need to provide it.